source/compatibility.tex

```diff
@@ -3720,7 +3720,9 @@
 The C headers \libheader{complex.h} and
 \libheader{tgmath.h} do not contain any of the content from
 the C standard library and instead merely include other headers from the \Cpp{}
-standard library.
+standard library. Whereas the \libheader{tgmath.h} macros can be suppressed in
+C by, for example, using parentheses around the macro name, the \Cpp{} overload
+sets cannot be bypassed in this manner; casts may be used instead.
```

Comment:
This seems tutorial-like to be honest. The wording is clear that the `tgmath.h` macros don't exist in C++, and that's all that really needs to be said about compatibility differences.

Even if we want to have this new sentence, it seems like something that goes into a `\begin{note}` context.

I'm also not sure what you mean by "casts may be used instead". Are you casting the function itself to `double(&)(double)`? That isn't valid because of addressability restrictions. You could cast all the function arguments like `sqrt((double)x)`, but that's possible in both languages, so it doesn't seem like it needs to be called out in [diff].

Reply:
> please tell us what you mean by those "casts".

Yes, I meant casting the arguments to match the C non-macro function signature.